### PR TITLE
Merging schedules app

### PR DESCRIPTION
**Pull request by alex-deaconu**
Updating routing
Various other fixes

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc I broke something and am working on a fix. Will let you know once the final version is ready. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Ok, no problem. Thanks!

**Comment by alex-deaconu** [comment]
@ezequielc e2e question is fixed. Please review. Thanks.

File: test/e2e/schedules/cases/add-distribution.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
+'use strict';
+var expect = require('rv-common-e2e').expect;
+var HomePage = require('./..//pages/homepage.js');
```
Comment (ezequielc) [inline test/e2e/schedules/cases/add-distribution.js]: @alex-deaconu Is there any particular reason for the double slash `//` in the requires? Thanks

Comment (alex-deaconu) [inline test/e2e/schedules/cases/add-distribution.js]: @ezequielc oops nice find. No it shouldn't be like that. Search & replace error. Fixed in subsequent commit. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu WOW! Switching from Home to Schedules is so fast now! Such a good feeling :)
PR looks good. :+1: Thanks!

**Pull request merged**